core/capabilities/launcher.go

```diff
@@ -246,15 +311,23 @@ func (w *launcher) Launch(ctx context.Context, state *registrysyncer.LocalRegist
 
 	// Finally, if I'm in a capability DON, let's enable external access
 	// to the capability.
-	if len(myCapabilityDONs) > 0 {
+	belongsToACapabilityDON := len(myCapabilityDONs) > 0
```

Comment:
I'm for expressive variable names, but seems redundant with the comment also explaining